### DISCUSSION TITLE
font.save function not working, not sure if bug

### DISCUSSION THREAD

**Discussion by devmgardner** [fonttools/fonttools#2568] [Q&A] [answered]
I'm trying to use fonttools to install .ttf files automatically, and it seems to not be working properly on Windows 10. I've tried as administrator as well. With "fontfile" as a full filepath to a .ttf file, the following code does nothing.

```
from fontTools.ttLib import TTFont
...
font = TTFont(fontfile)
font.save(fontfile, reorderTables=True)
```

**Comment by justvanrossum** [comment, marked as answer]
Firstly, I'm not sure why you'd want to use `font.save()` to "install".

The code you post opens a font file, then saves it onto itself. So in a way, your code indeed does nothing.

So: please be more clear about what you'd like to achieve, and explain what you expect "install ... automatically" to do.

**Comment by devmgardner** [comment]
Interesting. I must have misunderstood what the library is for, in that case; I understood it as "installing" a .ttf file, not as saving a .ttf file out. That makes a lot of sense! I've got quite a bit of work ahead of me then, as I'll have to find another way to install .ttf files through Python.

**Comment by justvanrossum** [comment]
The library is to parse / read / manipulate / write / create font files. It knows nothing about installing.

**Reply by devmgardner** [reply]
Got it, thank you!
